### PR TITLE
Formatting changes

### PR DESCRIPTION
**Pull request by wandrewkeech**
addresses the K's

### CODE REVIEW

File: minetest/snapcraft.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
- "infinite" map generator.
+description: |
+  Minetest is a minecraft-inspired game written from scratch and licensed
+  under the LGPL (version 2.1 or later). It supports both survival and
+  creative modes along with multiplayer support, dynamic lighting, and an
+  "infinite" map generator.
+confinement: devmode
 
 apps:
   minetest:
```
Comment (didrocks) [inline minetest/snapcraft.yaml]: On this part and following one: please put `source` before `plugins`

**Comment by tsimonq2** [comment]
Please remove `imagemagick-stable/imagemagick-stable_6.9.7_source.tar.bz2`

File: ubuntu-clock-app/snapcraft.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: ubuntu-clock-app
 version: 3.6+snap2
 summary: Ubuntu Clock app
 description: |
-   The clock app for all Ubuntu devices
+  The clock app for all Ubuntu devices
```
Comment (didrocks) [inline ubuntu-clock-app/snapcraft.yaml]: I guess that one can stay on the description: line and removing |

File: minetest/snapcraft.yaml [hunk 3]
```diff
@@ -17,15 +19,15 @@ parts:
...
-    plugin: copy
     source: https://github.com/minetest/minetest_game.git
+    plugin: copy
     files:
       '*': 'games/minetest_game/'
   minetest:
-    plugin: cmake
-    configflags: [-DRUN_IN_PLACE=TRUE]
     source: https://github.com/minetest/minetest.git
     source-type: git
```
Comment (tsimonq2) [inline minetest/snapcraft.yaml]: You should get rid of this

File: moon-buggy/snapcraft.yaml [hunk 4]
```diff
@@ -16,8 +16,8 @@ parts:
     build-packages:
       - libncurses5-dev
       - texinfo
-    plugin: autotools
     source: https://github.com/sergiusens/moon-buggy
     source-type: git
```
Comment (tsimonq2) [inline moon-buggy/snapcraft.yaml]: You should get rid of this

File: mpv/snapcraft.yaml [hunk 5]
```diff
@@ -25,17 +26,31 @@ parts:
     files:
       mpv: bin/mpv
   mpv:
-    plugin: waf
     source: https://github.com/mpv-player/mpv.git
     source-type: git
```
Comment (tsimonq2) [inline mpv/snapcraft.yaml]: You should get rid of this

**Comment by didrocks** [comment]
some changes (and conflicts against master):
- minetests & moon-buggy & tyrant: it doesn't follow the source, "source options", plugin order
- also, please remove source-type: git when possible (minetest, moon-buggy, mp, tyrant)
- mpv part is after ffmepg, shouldn't we declaret the ffmepg part first then?
- tyrant has a line break between the 2 apps. Also build-packages is before stage-packages
- ubuntukylin-icon-theme: plugs could all be in one line I think.
- ubuntukylin-icon-theme:
```
   source: lp:ubuntukylin-theme
   source-type: bzr
```
-> can that be autodetected? (lp: is bzr specific)
ubuntukylin-icon-theme: after: can be in one line IMHO
- ubuntukylin-icon-theme: I would put slots: before parts: (but isn't that plugs: btw?)

File: ubuntukylin-icon-theme/snapcraft.yaml [hunk 6]
```diff
@@ -35,9 +35,9 @@ apps:
 
 parts:
   theme:
-    plugin: copy_and_edit
     source: lp:ubuntukylin-theme
     source-type: bzr
```
Comment (tsimonq2) [inline ubuntukylin-icon-theme/snapcraft.yaml]: You should get rid of this

**Comment by tsimonq2** [comment]
I've actually looked at the detection code in Snapcraft for a PR I'm working on, and it does detect `lp:`.

**Comment by didrocks** [comment]
So, from the current state, I would see remaining to get fixed:
- moon-buggy: I would put `build-packages:` last here
- mpv (ffmpeg part): still has an `source-type`.
- ubuntu-clock-app: `configflags` after `plugin` and `source-type` is unecessary
- ubuntu-kylin-icon-theme: `slots:` is `plugs:` (even if I'm unsure the interface still exist) and it should be before `parts:`

**Comment by tsimonq2** [comment]
@wandrewkeech @didrocks :+1:

**Comment by didrocks** [comment]
LGTM as well, big thanks! Merging :)

**Pull request merged**